### PR TITLE
DTSAM-336: CASEWORKER_FTA_ENABLED and JUDICIAL_FTA_ENABLED flags.

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### Jira link (if applicable)

   [DTSAM-336](https://tools.hmcts.net/jira/browse/DTSAM-336) _"ORM FTAs may fail if Ref-Data services are unavailable. Add EV flag option to disable those FTAs if needed"_

### Change description ###

Added CASEWORKER_FTA_ENABLED and JUDICIAL_FTA_ENABLED flags.

> NB: test PRs:
> * #1944
> * #1945
> * #1946
> * #1947

### Checklist
- [x] Does this PR introduce a breaking change - No

### CODE REVIEW

**Review by mattnayler — APPROVED** [review]
LGTM

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -85,6 +85,10 @@ env.PACT_BROKER_URL = "pact-broker.platform.hmcts.net"
 env.PACT_BROKER_PORT = "443"
 env.PACT_BROKER_SCHEME = "https"
 
+// Turn off FTAs if RD is unavailable, DTSAM-336
+env.JUDICIAL_FTA_ENABLED = "false"
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply Jenkinsfile_CNP]: The jenkins scripts have been updated to include a comment pointing to ticket explaining why the JRD tests are currently **temporarily** disabled:
```
env.JUDICIAL_FTA_ENABLED = "false" // temporarily disabled DTSAM-334
```

**Review by abul-syed — APPROVED** [review]
lgtm

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Pull request merged**